### PR TITLE
Bumping cql-execution version to 2.0.0 beta.2.

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
Updating cql-execution dependency:
dependency libraries now properly use passed in parameter values even if a default value was declared in the main library

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: dist/browser.js [hunk 1]
```diff
@@ -1827,6 +1827,21 @@ module.exports = class Calculator {
     return bundle;
   }
 
+  /**
+   * To ensure the "Measure Parameter" override is applied to all Libraries.
+   * @param lib
+   */
```
Comment (serhii-ilin) [inline dist/browser.js:1833]: Do we need to keep that?

File: dist/browser.js [hunk 2]
```diff
@@ -15649,11 +15665,12 @@ var ParameterDef = /*#__PURE__*/function (_Expression) {
...
-        return ctx.parameters[this.name]; // If default type exists, execute the default type
+        return ctx.parameters[this.name]; // If the parent context contains the name, return that
+      } else if (ctx.getParentParameter(this.name) !== undefined) {
+        var parentParam = ctx.getParentParameter(this.name);
+        return parentParam.default != null ? parentParam.default.execute(ctx) : parentParam; // If default type exists, execute the default type
       } else if (this.default != null) {
-        return this.default != null ? this.default.execute(ctx) : undefined; // Else, if context and context's parent exist return the value of the parent's parameters with the given name.
-      } else {
-        return ctx.getParentParameter(this.name);
+        this.default.execute(ctx);
```
Comment (serhii-ilin) [inline dist/browser.js:15673]: doesn't looks like execution was browserified. is that expected?

File: dist/browser.js (hunk 1 above)
Comment (jkotanchik-SB) [reply dist/browser.js:1833]: Not at all. I commented out the call, but not the function and didn't realize it'd be included in the browser.js.

File: dist/browser.js (hunk 2 above)
Comment (jkotanchik-SB) [reply dist/browser.js]: It appears similar to the current contents, but perhaps it never worked correctly. 

@AndrewBird81 Does this look like accurate browserified content?

File: package.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
   "dependencies": {
     "bson-objectid": "^1.3.1",
     "cql-exec-fhir": "^1.5.0-beta.1",
```
Comment (adongare) [inline package.json:67]: How about `cql-exec-fhir`?  this has cql-execution dependency as well. Is that needs to be updated as well?

Comment (jkotanchik-SB) [reply package.json:67]: It's a dev dependency on `cql-exec-fhir` so it shouldn't impact our usage.

**Review by adongare — APPROVED** [review]

**Review by serhii-ilin — APPROVED** [review]
